client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/style.scss

```
@@ -54,15 +54,14 @@ main.business-plan-upgrade-upsell-new-design.main {
 	}
 
 	&__image-container {
-		min-width: 200px;
 		grid-area: images;
 		text-align: center;
 		@include breakpoint-deprecated( "<660px" ) {
 			padding: 20px 0;
 		}
 
 		img {
-			max-width: 60%;
+			margin: 0 0 0 30px;
```

On mobile, the image does not look centered, with this it could fixed

```suggestion
			@include breakpoint-deprecated( ">660px" ) {
				margin: 0 0 0 30px;
			}
```

---

This did the trick 👍